lagom-cloud-object-storage-example/README.md

```diff
@@ -0,0 +1,108 @@
+    ```
+2.  Change into the root directory for this example:
+    ```
+    cd lagom-cloud-object-storage-example
+    ```
+3.  Supply the configuration:
+    1. Copy the `account-impl/src/main/resources/cloud-object-storage.conf.template` file to `account-impl/src/main/resources/cloud-object-storage.conf`.
+    2. Open `account-impl/src/main/resources/cloud-object-storage.conf` in a text editor and fill in the necessary information. Details are provided in the file itself.
+
+## Start the Lagom sample application
+
+In the command line shell where you downloaded the Lagom service, from the `lagom-cloud-object-storage-example` directory, start the Lagom development environment by running:
+
+```
+mvn lagom:runAll
+```
+
+You should see some console output, including these lines:
+
+```
+...
+[INFO] Service gateway is running at http://localhost:9000
+...
+[INFO] (Service started, press enter to stop and go back to the console...)
+```
+
+These messages indicate that the service has started correctly.
+
+
+## Generate some traffic on the Lagom service
+
+From a new terminal, you should now generate some traffic on the Lagom service. 
+
+This application simulates a bank account application where deposits and withdraws can be executed on an account. On every 5 transactions an account extract is generated and uploaded to Cloud Object Storage. Extracts can be downloaded for local visualization. 
+
+Account transactions are propagated from the write-side (Account Entity) to the read-side (Extract Processor) as events stored in a Cassandra database. 
+
+Extracts are kept in-memory until they reach 5 transactions when they are uploaded to Cloud Object Storage. For the sake of simplicity, the `AccountExtractRepository`, that holds extracts in-memory, is not thread-safe and therefore its code is only suitable for demonstrations.
+
+This application doesn't have a GUI. Only a REST API. Any REST client or http tool can be used to interact with it. We provide a simple `api.sh` that can be used as a shell client that uses [HTTPie](https://httpie.org/).  
+
+  1. Source `api.sh` in your console (eg: `. api.sh`) - optional 
+  2. Call   
+     ```
+     account.deposit 123-4567-890 100  
+     # or equivalent curl command
+     `curl -H "Content-Type: application/json" -XPOST http://localhost:9000/api/account/123-4567-890/deposit --data '{ "amount": 100 }'
+     ```
+  3. Call   
+     ```
+     account.withdraw 123-4567-890 100
+     # or equivalent curl command  
+     curl -H "Content-Type: application/json" -XPOST http://localhost:9000/api/account/123-4567-890/withdraw --data '{ "amount": 100 }'
+     ```
+  4. Repeat step 2 and 3 a couple of times. Watch out to not withdraw more than your current balance.  
+     You can check the balance by calling.  
+     ```
+     account.balance 123-4567-890 100
+     # or equivalent curl command  
+     curl http://localhost:9000/api/account/123-4567-890/balance
+     ```
```

Comment:
yep, will reorganise it.

Reply:
Maybe the best here is explain the API calls with only curl as example.
 
Because we have already `api.sh` and HTTPie is really handy. We can keep it and add docs in the `api.sh`. We could mention it and invite user to read the docs there if they prefer.